borg/archive.py

```diff
@@ -786,11 +791,11 @@ def add_callback(chunk):
 
         def add_reference(id_, size, csize, cdata=None):
             try:
-                count, _, _ = self.chunks[id_]
-                self.chunks[id_] = count + 1, size, csize
```

Comment:
as it does more stuff than this on a KeyError now, using .get() seems no option here.

yes, the assertion might be badly placed, have to investigate.